src/Console/Application.php

```diff
@@ -163,6 +163,10 @@ protected function configureIO(InputInterface $input, OutputInterface $output):
         $output->getFormatter()->setStyle('low', new OutputFormatterStyle('red', null, ['bold']));
         $output->getFormatter()->setStyle('medium', new OutputFormatterStyle('yellow', null, ['bold']));
         $output->getFormatter()->setStyle('high', new OutputFormatterStyle('green', null, ['bold']));
+
+        if ($input->isInteractive() && (getenv('CI') === 'true' || getenv('CONTINUOUS_INTEGRATION') === 'true')) {
```

Comment:
Perhaps it makes sense to use the `BuildContextResolver` here?

After all, when a `BuildContext` could be resolved, we must be running in a continuous integration environment.

Reply:
Ha, looking at #1067, that's not entirely true.